semanticdb/semanticdb3/guide.md

```diff
@@ -41,9 +41,9 @@ Make sure you are using the latest coursier version (1.1.0-M4 or newer).
 [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.scalameta/scalameta_2.12/badge.svg)](https://maven-badges.herokuapp.com/maven-central/org.scalameta/scalameta_2.12)
 
 ```bash
-alias metac="coursier launch org.scalameta:metac_2.12.6:4.0.0-M4 -- -cp $(coursier fetch -p org.scala-lang:scala-library:2.12.6)"
-alias metacp="coursier launch org.scalameta:metacp_2.12:4.0.0-M4 --"
-alias metap="coursier launch org.scalameta:metap_2.11:4.0.0-M4 --"
+alias metac="coursier launch org.scalameta:metac_2.12.6:4.0.0-M5 -- -cp $(coursier fetch -p org.scala-lang:scala-library:2.12.6)"
+alias metacp="coursier launch org.scalameta:metacp_2.12:4.0.0-M5 --"
```

Comment:
This command will cause problems since --dependency-classpath is missing. Metacp doesn't even infer JDK jars by default. I think we can improve on metacp cli ergonomics.